src/virt.c

```diff
@@ -130,13 +133,22 @@ enum plginst_field {
 static enum plginst_field plugin_instance_format[PLGINST_MAX_FIELDS] =
     { plginst_none };
 
+/* BlockDeviceFormat */
+enum bd_field {
+	dev,
+	source
+};
+
 /* InterfaceFormat. */
 enum if_field {
     if_address,
     if_name,
     if_number
 };
 
+/* BlockDeviceFormatBasename */
+char *blockdevice_format_basename="false";
```

Comment:
Please mimic the style of the rest of the file, e.g. spaces around the equal sign.